verify-service-provider.yml

```diff
@@ -55,6 +58,9 @@ samlPrimaryEncryptionKey: ${SAML_PRIMARY_ENCRYPTION_KEY:-}
 # This only needs to be set if during key rotations (for example if your primary encryption certificate is about to expire)
 samlSecondaryEncryptionKey: ${SAML_SECONDARY_ENCRYPTION_KEY:-}
 
+# Enable European identities
+# Setting `europeanIdentity.enabled: true` allows your VSP to consume European
+# identities.
```

Comment:
Please note that the whole section is now optional.  If it is omitted, it will not be possible to process eIDAS identities.  Likewise, if the section is present but the 'enabled' setting is false, the other settings will not be used (although if they are present, their values must be valid) and it will not be possible to process eIDAS identities.

Reply:
I see. I've re-organised comments to make this clearer :)